Documentation/BestPractises/CommonFields.rst

```diff
@@ -16,11 +18,14 @@ If the table has a TCA definition, TYPO3 will automatically create the following
     and as a reference in relationships between records.
 
 :sql:`pid`
-    The `uid` property of the parent page. The record is situated on this page. This value is 0 if the record is not connected to any page.
+    The `uid` property of the parent page. The record is situated on this page.
```

Comment:
I know, it was "property" before, but I would prefer to use "property" in extbase context only. For TCA I prefer "column" (of the table) or "field" in BE form.

```suggestion
    The `uid` field of the parent page. The record is situated on this page.
```